docs/source/reference/static_graph.rst

```diff
@@ -1,9 +1,9 @@
 Static Subgraph Optimizations: Usage
-====================================================
```

Comment:
How about splitting the changes to this file?
I want to backport it.

Reply:
Separated into https://github.com/chainer/chainer/pull/7389.